ansys/dpf/core/_version.py

```diff
@@ -1,8 +1,8 @@
 """Version for ansys-dpf-core"""
 # major, minor, patch
-version_info = 0, 3, "dev0"
+version_info = 0, 4, '0'
```

Comment:
Version tag should be `"dev0"` on `master` branch. Non dev versions should be on the release branches.